plugins/migrator/params.go

```diff
@@ -0,0 +1,28 @@
+package migrator
+
+import (
+	"time"
+
+	"github.com/gohornet/hornet/pkg/node"
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	// CfgMigratorStateFilePath configures the path to the state file of the migrator.
+	CfgMigratorStateFilePath = "migrator.stateFilePath"
+	// CfgMigratorQueryCooldownPeriod configures the cooldown period for the service to ask for new data
+	// from the legacy node in case the migrator encounters an error.
+	CfgMigratorQueryCooldownPeriod = "migrator.queryCooldownPeriod"
```

Comment:
In seconds? what is the unit here?

Reply:
Ignore Alex comment here, because I have another PR opened, which removes all "Seconds" etc from the config settings, and uses the `Duration` notation you used here.